code/modules/clothing/under/jobs/medsci.dm

```diff
@@ -161,8 +161,18 @@
 /obj/item/clothing/under/rank/medical/paramedic
 	desc = "It's made of a special fiber that provides minor protection against biohazards. It has a dark blue cross on the chest denoting that the wearer is a trained paramedic."
 	name = "paramedic jumpsuit"
-	icon_state = "paramedic"
+	icon_state = "paramedic-dark"
 	item_state = "w_suit"
+	item_color = "paramedic-dark"
+	permeability_coefficient = 0.5
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 10, "rad" = 0, "fire" = 0, "acid" = 0)
+
+/obj/item/clothing/under/rank/medical/paramedic/light
+	desc = "It's made of a special fiber that provides minor protection against biohazards. It has a dark blue cross on the chest denoting that the wearer is a trained paramedic."
+	name = "paramedic jumpsuit"
```

Comment:
redundant variable, change to "light paramedic jumpsuit" or just remove.

Reply:
same for the desc.

Reply:
actually, pretty much every var here is redundant except for the icon_state and item_state.

Reply:
the name is in fact redundant but the description is not